lib/react_on_rails/configuration.rb

```diff
@@ -266,6 +266,14 @@ def raise_missing_components_subdirectory
 
       raise ReactOnRails::Error, msg
     end
+
+    def shakapacker_precompile?
+      config = Webpacker.config
```

Comment:
In the future, won't Webpacker be unavailable?

Reply:
Sure.
But still, we need to provide support for the old spelling.

As soon as we force the upgrade to Shakapacker 7+, we need to have support for webpacker spelling.

Reply:
At some point, we release an update to Shakapacker.

Somebody has not yet updated React on Rails.

`config = Webpacker.config` crashes.

Check that the constant `Webpacker` exists before calling it.

Other code spots might have the same issue. Check.

Reply:
Let's fix this issue in another dedicated PR.

I think at some point, we should drop support Shakapacker 6. I think it is important to push projects to move to use the new API for shakapacker and version 7 was the transitional version for that migration.